master/buildbot/worker/openstack.py

```diff
@@ -207,7 +207,10 @@ def _determineVolumeSize(self, source_type, source_uuid):
         if source_type == 'image':
             # The size returned for an image is in bytes. Round up to the next
             # integer GiB.
-            image = nova.images.get(source_uuid)
+            if hasattr(nova, "glance"):
+                image = nova.glance.get(source_uuid)
+            else:
```

Comment:
Probably safe to delete this. Nova removed this part a long time ago (novaclient 8.0 I think?)

Reply:
I feel bit bad about removing it. I know it's ages ago, but then again, it's very small tweak to maintain compability with ancient openstack.